src/state/slices/accountSpecifiersSlice/accountSpecifiersSlice.ts

```diff
@@ -27,11 +27,11 @@ export const accountSpecifiers = createSlice({
       moduleLogger.info('clearing account specifiers')
       return getInitialState()
     },
-    setAccountSpecifiers(state, { payload }: { payload: AccountSpecifierMap[] }) {
+    upsertAccountSpecifiers(state, { payload }: { payload: AccountSpecifierMap[] }) {
```

Comment:
only risky parts of this PR. previously these were drop & insert style setters, now upserters.

this allows us to load the portfolio with account number 0, then upsert account n's later